demo-apps/angular/angular-app/src/app/app.component.html

```diff
@@ -0,0 +1,27 @@
+
+<style>
+
+  main {
+    width: 100%;
+    min-height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem;
+    box-sizing: inherit;
+    position: relative;
+  }
+
+
+</style>
+
+<gcds-container id="main" size="xl" tag="main" main-container centered>
+
```

Comment:
Very nit-picky I know, but can we remove the extra empty spaces after container and text? :D

Reply:
Done!